draft-ietf-webtrans-http3.md

```diff
@@ -396,11 +401,10 @@ application errors and HTTP/3 error codes" }
 
 WebTransport data streams are associated with sessions through a header at the
 beginning of the stream; resetting a stream may result in that data being
-discarded.  Because of that, WebTransport application error codes are best
-effort, as the WebTransport stack is not always capable of associating the
-reset code with a session.  The only exception is the situation where there is
-only one session on a given HTTP/3 connection, and no intermediaries between
-the client and the server.
+discarded.  Because of that, WebTransport implementations MUST use the
+RESET_STREAM_AT frame {{RESET-STREAM-AT}} when resetting WebTransport data
+streams to ensure that the stream header containing the associated WebTransport
+session ID is always delivered.
```

Comment:
Suggestion (GitHub doesn't allow me to mark this a suggestion since it covers unchanged lines):
> WebTransport data streams are associated with sessions through a header at the
beginning of the stream; resetting a stream might result in that data being
discarded when using a RESET_STREAM frame.
> To prevent this, WebTransport implementations MUST use the RESET_STREAM_AT frame {{RESET-STREAM-AT}} with a Reliable Size set to at least the size of the header when resetting WebTransport data. This ensure that the stream header containing the associated WebTransport session ID is always delivered.

Reply:
Thanks! Adopted this with a few modifications.